binary_search_tree/binary_search_tree.py

```diff
@@ -42,8 +42,15 @@ def inOrder(self):
     if self.right != None:
       self.right.inOrder()
 
+  def preOrder(self):
+    # print(self.value)
+    if self.left != None:
+      self.left.inOrder()
+    if self.right != None:
+      self.right.preOrder()
+
   def get_max(self):
-    if self.right == None:
-      return self.value
+    if self.right != None:
+      return self.right.get_max()
```

Comment:
very nice! yep, if there's no more values to the right of a node in a BST, it means that that node is the largest. awesome job man!